platform.txt

```diff
@@ -129,3 +129,27 @@ tools.openocd.erase.pattern=
 tools.openocd.bootloader.params.verbose=-d2
 tools.openocd.bootloader.params.quiet=-d0
 tools.openocd.bootloader.pattern="{path}/{cmd}" {bootloader.verbose} -f interface/{program.protocol}.cfg -c "{program.setup_command}" -f target/{upload.target}.cfg -c "init; halt; nrf51 mass_erase; program {{{runtime.platform.path}/cores/nRF5/SDK/components/softdevice/{softdevice}/hex/{softdevice}_{upload.target}_{softdeviceversion}_softdevice.hex}} verify reset; shutdown;"
+
+#
+# blackmagic probe upload
+#
+tools.blackmagicprobe.path={runtime.tools.gcc-arm-none-eabi-5_2-2015q4.path}/bin/
```

Comment:
Can you replace the `{runtime.tools.gcc-arm-none-eabi-5_2-2015q4.path}/bin/` with `{compiler.path}` here?

That way if gcc is upgraded, only one line needs to be updated:
```
compiler.path={runtime.tools.gcc-arm-none-eabi-5_2-2015q4.path}/bin/
```

Reply:
unfortunately it looks like {compiler.path} is not visible to tools, only to the compiler itself and recipes.

Reply:
Confirmed.